packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -494,13 +511,8 @@ class AttachDisk extends Component {
                 {_('vbdAttach')}
               </ActionButton>
             </span>
-            {!this._checkSr() && (
-              <div>
-                <span className='text-danger'>
-                  <Icon icon='alarm' /> {_('warningVdiSr')}
-                </span>
-              </div>
-            )}
+            {this._checkVdi() && Warning(_('warningAttachedVdi'))}
```

Comment:
Syntactically, if the VDI check is true it means that it have no problems with the selected VDI, no?